### PR TITLE
STY: Fix bad style in umath_linalg

### PR DESCRIPTION
**Pull request by eric-wieser**
Raised by #8756

### CODE REVIEW

File: numpy/linalg/umath_linalg.c.src [hunk 1]
```diff
@@ -1605,13 +1608,13 @@ typedef struct gesv_params_struct
 } GESV_PARAMS_t;
 
 /**begin repeat
-   #TYPE=FLOAT,DOUBLE,CFLOAT,CDOUBLE#
-   #typ=npy_float,npy_double,npy_cfloat,npy_cdouble#
-   #ftyp=fortran_real,fortran_doublereal,fortran_complex,fortran_doublecomplex#
-   #lapack_func=sgesv,dgesv,cgesv,zgesv#
+   #TYPE = FLOAT, DOUBLE, CFLOAT, CDOUBLE#
+   #typ = npy_float, npy_double, npy_cfloat, npy_cdouble#
+   #ftyp = fortran_real, fortran_doublereal, fortran_complex, fortran_doublecomplex#
```
Comment (charris) [inline numpy/linalg/umath_linalg.c.src]: You can break long lines, grep for examples.

Comment (eric-wieser) [inline numpy/linalg/umath_linalg.c.src]: Wasn't paying attention to the fact this was long - oops

File: numpy/linalg/umath_linalg.c.src [hunk 2]
```diff
@@ -368,21 +368,21 @@ typedef f2c_doublecomplex fortran_doublecomplex;
...
 offset_ptr(void* ptr, ptrdiff_t offset)
 {
     return (void*)((npy_uint8*)ptr + offset);
 }
 
-static inline int
+static NPY_INLINE int
 get_fp_invalid_and_clear(void)
 {
     int status;
```
Comment (charris) [inline numpy/linalg/umath_linalg.c.src]: Need blank line after declaration.

File: numpy/linalg/umath_linalg.c.src [hunk 3]
```diff
@@ -651,39 +651,39 @@ CDOUBLE_mulc(DOUBLECOMPLEX_t op1, DOUBLECOMPLEX_t op2)
...
+static NPY_INLINE void
 print_DOUBLE(npy_double d)
 {
     TRACE_TXT(" %10.6f", d);
 }
-static inline void
+static NPY_INLINE void
 print_CFLOAT(npy_cfloat c)
 {
     float* c_parts = (float*)&c;
```
Comment (charris) [inline numpy/linalg/umath_linalg.c.src]: Blank line after declaration. Also in other spots.

File: numpy/linalg/umath_linalg.c.src [hunk 4]
```diff
@@ -912,12 +912,12 @@ delinearize_@TYPE@_matrix(void *dst_in,
...
 
-static inline void
+static NPY_INLINE void
 nan_@TYPE@_matrix(void *dst_in, const LINEARIZE_DATA_t* data)
 {
     @typ@ *dst = (@typ@ *) dst_in;
 
-    int i,j;
+    int i, j;
     for (i=0; i < data->rows; i++) {
```
Comment (charris) [inline numpy/linalg/umath_linalg.c.src]: Space around `=`.

File: numpy/linalg/umath_linalg.c.src [hunk 5]
```diff
@@ -956,15 +956,15 @@ identity_@TYPE@_matrix(void *ptr, size_t n)
...
 
-static inline void
+static NPY_INLINE void
 triu_@TYPE@_matrix(void *ptr, size_t n)
 {
-    size_t i,j;
+    size_t i, j;
     @typ@ *matrix = (@typ@*)ptr;
     matrix += n;
     for (i=1; i < n; ++i) {
```
Comment (charris) [inline numpy/linalg/umath_linalg.c.src]: Space around `=`, also blank line after declarations.

File: numpy/linalg/umath_linalg.c.src [hunk 6]
```diff
@@ -1111,8 +1111,7 @@ static inline void
     /* note: done in place */
     LAPACK(@cblas_type@getrf)(&m, &m, (void *)src, &lda, pivots, &info);
 
-    if (info == 0)
-    {
+    if (info == 0) {
         int change_sign = 0;
         /* note: fortran uses 1 based indexing */
         for (i=0; i < m; i++)
```
Comment (charris) [inline numpy/linalg/umath_linalg.c.src]: Space around `=`. Might as well search for that.

**Comment by charris** [comment]
Much improved, still a few more things to take care of.

File: numpy/linalg/umath_linalg.c.src (hunk 3 above)
Comment (eric-wieser) [inline numpy/linalg/umath_linalg.c.src]: Too much work to find, I think. If we really care about that, then something like `astyle` might be a good plan

File: numpy/linalg/umath_linalg.c.src [hunk 7]
```diff
@@ -3154,7 +3179,7 @@ GUFUNC_DESCRIPTOR_t gufunc_descriptors [] = {
     {
         "svd_m_f",
         "(m,n)->(m,m),(m),(n,n)",
-        "svd when m>=n",
+        "svd when m<=n",
```
Comment (eric-wieser) [inline numpy/linalg/umath_linalg.c.src]: Copy and paste error. The rule is that the single letter in `()` is the minimum of `m` and `n`

File: numpy/linalg/umath_linalg.c.src (hunk 3 above)
Comment (charris) [inline numpy/linalg/umath_linalg.c.src]: Yes, we should be able to automate things to some extent. There are some decent tools out there. However, once done it isn't too difficult to keep things from sliding.

Comment (eric-wieser) [inline numpy/linalg/umath_linalg.c.src]: Either way, I think I plan to leave this PR at this point (modulo a squash, perhaps). If someone else wants to push some extra cleanup to this branch, they're welcome to. Or maybe it's worth merging as is.

File: numpy/linalg/umath_linalg.c.src [hunk 8]
```diff
@@ -706,15 +706,15 @@ dump_@TYPE@_matrix(const char* name,
  *****************************************************************************
  */
 
-static inline fortran_int
+static NPY_INLINE fortran_int
 fortran_int_min(fortran_int x, fortran_int y) {
     if (x < y)
```
Comment (charris) [inline numpy/linalg/umath_linalg.c.src]: Why not `return (x < y) ? x : y;`

Comment (eric-wieser) [inline numpy/linalg/umath_linalg.c.src]: Less benefit to a ternary if not in a macro, I guess. Want me to change that?

Comment (charris) [inline numpy/linalg/umath_linalg.c.src]: It would keep me from complaining about missing `{}` ;)

Comment (eric-wieser) [inline numpy/linalg/umath_linalg.c.src]: Done

**Comment by charris** [comment]
Thanks Eric.

**Pull request merged**